app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html

```diff
@@ -0,0 +1,1634 @@
+  </p>
+  <p class="govuk-body">
+    Mae'n rhaid i hwn gyfateb i enw cyfreithiol cofrestredig eich sefydliad
+  </p>
+  <p class="govuk-body">
+    A yw eich sefydliad yn defnyddio unrhyw enwau eraill?
+  </p>
+  <p class="govuk-body">
+    Os 'Ydy', ewch i <strong>1.1.1.1</strong>
+  </p>
+  <p class="govuk-body">
+    Os 'Nac ydy', ewch i <strong>1.1.2</strong>
+  </p>
+  <h4 class="govuk-heading-s">
+    1.1.1.1. Enwau amgen eich sefydliad
+  </h4>
+  <p class="govuk-body">
+    Enwau amgen eich sefydliad
+  </p>
+  <p class="govuk-body">
+    Enw amgen 1
+  </p>
+  <p class="govuk-body">
+    Enw amgen 2 (dewisol)
+  </p>
+  <p class="govuk-body">
+    Enw amgen 3 (dewisol)
+  </p>
+  <h4 class="govuk-heading-s">
+    1.1.2. Diben a gweithgareddau
+  </h4>
+  <p class="govuk-body">
+    Beth yw prif ddiben eich sefydliad?
+  </p>
+  <p class="govuk-body">
+    Dyma beth y sefydlwyd y sefydliad i'w gyflawni.
+  </p>
+  <p class="govuk-body">
+    (Max 500 words)
+  </p>
+  <p class="govuk-body">
+    Dywedwch wrthym am brif weithgareddau eich sefydliad
+  </p>
+  <p class="govuk-body">
+    Cynhwyswch unrhyw weithgareddau yr ydych yn ymgymryd Ã¢ nhw er mwyn cyflawni pwrpas y sefydliad.
+  </p>
+  <p class="govuk-body">
+    Rhaid i chi restru o leiaf un, a gall gynnwys hyd at dri.
+  </p>
+  <p class="govuk-body">
+    Gweithgarwch 1
+  </p>
+  <p class="govuk-body">
+    (Max 500 words)
+  </p>
+  <p class="govuk-body">
+    Gweithgarwch 2 (dewisol)
+  </p>
+  <p class="govuk-body">
+    (Max 500 words)
+  </p>
+  <p class="govuk-body">
+    Gweithgarwch 3 (dewisol)
+  </p>
+  <p class="govuk-body">
+    (Max 500 words)
+  </p>
+  <p class="govuk-body">
+    A ydych chi wedi cyflawni prosiectau fel hwn o'r blaen?
+  </p>
+  <p class="govuk-body">
+    Os 'Ydy', ewch i <strong>1.1.2.1</strong>
+  </p>
+  <p class="govuk-body">
+    Os 'Nac ydy', ewch i <strong>1.1.3</strong>
+  </p>
+  <h4 class="govuk-heading-s">
+    1.1.2.1. Prosiectau blaenorol sy'n debyg i'r prosiect hwn
+  </h4>
+  <p class="govuk-body">
+    Disgrifiwch eich prosiectau blaenorol
+  </p>
+  <p class="govuk-body">
+    Prosiect 1
+  </p>
+  <p class="govuk-body">
+    (Max 500 words)
+  </p>
+  <p class="govuk-body">
+    Prosiect 2 (dewisol)
+  </p>
+  <p class="govuk-body">
+    (Max 500 words)
+  </p>
+  <p class="govuk-body">
+    Prosiect 3 (dewisol)
+  </p>
+  <p class="govuk-body">
+    (Max 500 words)
+  </p>
+  <h4 class="govuk-heading-s">
+    1.1.3. Sut mae eich sefydliad yn cael ei ddosbarthu
+  </h4>
+  <p class="govuk-body">
+    Math o sefydliad
+  </p>
+  <p class="govuk-body">
+    Dewiswch un opsiwn
+  </p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li class="">
+      Sefydliad corfforedig elusennol (CIO)
+    </li>
+    <li class="">
+      Cwmni cydweithredol, fel cymdeithas budd cymunedol
+    </li>
+    <li class="">
+      Cwmni buddiannau cymunedol (CIC)
+    </li>
+    <li class="">
+      Company limited by guarantee
```

Comment:
This is displaying in English.

Reply:
This is another missing translation which we need to make a ticket for

Reply:
https://dluhcdigital.atlassian.net/browse/FS-3862 ticket created.